softwarereview_editor.Rmd

```diff
@@ -126,48 +121,54 @@ Each submission should be reviewed by *two* package reviewers. Although it is fi
 
 ## EiC Responsibilities {#eicchecklist}
 
-The EiC serves for 3 months or a time agreed to by all members of the editorial board.
-The EiC is entitled to taking scope and overlap decisions as independently as possible (but can still request help/advice).
-In details, the EiC plays the following roles
+Rotating Editors-in-Chief (EiCs) generally serve for 3 months or a time agreed to by all members of the editorial board.
+The EiC is entitled to taking scope and overlap decisions as independently as possible (but can still request help and advice).
```

Comment:
why do we use the acronym instead of writing it in full? genuine question

Reply:
Brevity. Because it is used 10 times, it does lessen the visual parsing burden here, I'd say

Reply:
Ok, makes sense